modules/apps/wiki/wiki-web/src/main/resources/META-INF/resources/wiki/view.jsp

```diff
@@ -307,10 +296,20 @@ if (portletTitleBasedNavigation) {
 
 							<liferay-ui:icon
 								icon="print"
+								id="printPageButton"
 								label="<%= true %>"
 								markupView="lexicon"
 								message="print"
-								url='<%= "javascript:" + liferayPortletResponse.getNamespace() + "printPage();" %>'
+								url="javascript:void(0);"
+							/>
+
+							<liferay-frontend:component
+								context='<%=
+									HashMapBuilder.<String, Object>put(
+										"printPageURL", printPageURL
+									).build()
+								%>'
+								module="wiki/js/printPageButtonPropsTransformer"
```

Comment:
I probably wouldn't call this a propsTransformer here because it isn't transforming any props. It can just be something like `printPageButtonEventListener.js`